pcvalidate/pcvalidate.go

```diff
@@ -94,8 +94,12 @@ func main() {
 	} else {
 		if err != nil {
 			fmt.Println(err)
+		}
+		if hasValidationErrors(err) {
 			os.Exit(1)
```

Comment:
Can we return the error instead of exiting brutally here?

Reply:
This is a command line tool, once we get there we already printed everything we needed and we have to exit (with status)

Am I missing something?